src/coffee/export.coffee

```diff
@@ -99,15 +108,118 @@ class Export
     new ExportMapping(@options)
 
   # return the correct product service in case query string is used or not
-  _getProductService: (staged = true) ->
+  _getProductService: (staged = true, customCondition = false) ->
```

Comment:
can we rename this to `customWherePredicate`? I think `customCondition` is too ambiguous.